protocols/v2/noise-sv2/src/responder.rs

```diff
@@ -1,3 +1,39 @@
+// # Responder Role
```

Comment:
responder is a public struct(exported from `lib`)

Reply:
We are currently exporting only the Responder struct, rather than the entire module.

Reply:
exactly, these header comments would only be shown if the module was public

Reply:
hmm yea, maybe that's something for the refactoring. `responder.rs` does not have anything in it beside the `Responder` struct. It is also a very important(required) struct, it would be beneficial for the library users to access all its docs easily on docs.rs

Reply:
That makes sense to me as well. During the refactoring, we can expose the entire responder.rs and initiator.rs files, along with the module documentation.